pinot-common/src/main/proto/worker.proto

```diff
@@ -70,23 +70,17 @@ message QueryResponse {
 
 message StagePlan {
   int32 stageId = 1;
-  string instanceId = 2;
+  string virtualAddress = 2;
```

Comment:
- we will keep virtualAddress here indicating the host/port and workerID together, cuz to a stage plan this will always be the same and non-nullable. 
- for mailbox/opchain decoupling purpose we will use mailboxID as a unique identifier generate from planner going forward and will be put into StageMetadata/WorkerMetadata